api/src/main/java/gov/cms/ab2d/api/controller/BulkDataAccessAPI.java

```diff
@@ -222,19 +229,26 @@ private String getUrlPath(Job job, String filePath) {
     )
     @ResponseStatus(value = HttpStatus.OK)
     @GetMapping(value = "/Job/{jobUuid}/file/{filename}")
-    public ResponseEntity<Resource> downloadFile(
+    public void downloadFile(
             @ApiParam(value = "A job identifier", required = true) @PathVariable @NotBlank String jobUuid,
             @ApiParam(value = "A file name", required = true) @PathVariable @NotBlank String filename) throws IOException {
         MDC.put(JOB_LOG, jobUuid);
         MDC.put(FILE_LOG, filename);
         log.info("Request submitted to download file");
 
         Resource downloadResource = jobService.getResourceForJob(jobUuid, filename);
-        HttpHeaders headers = new HttpHeaders();
-        headers.set(HttpHeaders.CONTENT_TYPE, NDJSON_FIRE_CONTENT_TYPE);
+        RequestAttributes requestAttributes = RequestContextHolder.getRequestAttributes();
+        HttpServletResponse response = ((ServletRequestAttributes) requestAttributes).getResponse();
+        response.setContentType(NDJSON_FIRE_CONTENT_TYPE);
+        OutputStream out = response.getOutputStream();
+        FileInputStream in = new FileInputStream(downloadResource.getFile());
+
+        IOUtils.copy(in, out);
 
         log.info("Sending file to client");
 
-        return new ResponseEntity<>(downloadResource, headers, HttpStatus.OK);
+        out.close();
+        in.close();
```

Comment:
Let's use a `try-with-resources` statement to make sure both streams get closed in case of an exception. Helps to avoid having to call `close` explicitly, too.